rSync_data_with_dataCenter/rSyncBackups.sh

```diff
@@ -68,7 +68,7 @@ echo -e "| Backup of files to host $red$bold_on$REMOTE_HOST$bold_off$color_off b
 echo -e "---------------------------------------------------------------------------------"
 logger "[rSyncBackups] Backup of files ( $FROM_DIR ) to host $REMOTE_HOST begins at `date`"
 
-/usr/bin/rsync -zh -e "ssh -o StrictHostKeyChecking=no -i $SSH_KEY " $FROM_DIR $REMOTE_USER@$REMOTE_HOST:$REMOTE_PARENT_DIR 2>/dev/null
+/usr/bin/rsync -zah -e "ssh -o StrictHostKeyChecking=no -i $SSH_KEY " $FROM_DIR $REMOTE_USER@$REMOTE_HOST:$REMOTE_PARENT_DIR 2>/dev/null
```

Comment:
Yes you are absolutely right!

But, `-r` or `-a` ?
```suggestion
/usr/bin/rsync -zrh -e "ssh -o StrictHostKeyChecking=no -i $SSH_KEY " $FROM_DIR $REMOTE_USER@$REMOTE_HOST:$REMOTE_PARENT_DIR 2>/dev/null
```

```
./rSyncBackups.sh -u root -r 123.456.789.0 -k ~/.ssh/publicKey -f test1.txt:test2.text:dataFolder -t /root/backup
```

* results on the remote server with `a` parameter :

```
.
└── backup
    ├── dataFolder
    ├── remote
    ├── test1.txt
    └── test2.text
```

* results on the remote server with `r` parameter :
```
.
└── backup
    ├── dataFolder
    ├── remote
    ├── test1.txt
    └── test2.text
```

-------------------------

```
./rSyncBackups.sh -u root -r 83.212.77.217 -k ~/.ssh/okeanos-Knossos -f test1.txt:test2.text:dataFolder/ -t /root/backup
```

-------------------------

`man rsync` :
* 	`-a, --archive               archive mode; equals -rlptgoD (no -H,-A,-X)`
* 	`-r, --recursive             recurse into directories`

Reply:
-a option includes -r (-a == -rlptgoD)
The other options is to preserve extra info (permission, info, groups, copy symlinks as is)
-r, --recursive recurse into directories
-l, --links copy symlinks as symlinks
-p, --perms preserve permissions
-t, --times preserve modification times
-g, --group preserve group
-o, --owner preserve owner (super-user only)
-D same as --devices --specials
--devices preserve device files (super-user only)
--specials preserve special files

I would say it better to use -a in my opinion

Reply:
Very nice!